stl/inc/vector

```diff
@@ -46,6 +46,7 @@ public:
 #if _ITERATOR_DEBUG_LEVEL != 0
         const auto _Mycont = static_cast<const _Myvec*>(this->_Getcont());
         _STL_VERIFY(_Ptr, "can't dereference value-initialized vector iterator");
+        _STL_VERIFY(_Mycont, "can't dereference invalidated vector iterator");

```

Comment:
Doesn't this subsume the previous line? Can we just check for `_STL_VERIFY(_Mycont` and not check for `_Ptr` as well?

Reply:
Well, we _could_ but I think there is a useful distinction to retain between attempting to dereference a _value initialized_ vector iterator and attempting to dereference an _invalided_ iterator.

Reply:
Ah, and additionally I believe `_Mycont` is only cleared at `_ITERATOR_DEBUG_LEVEL` of 2, so the check would be lost at level 1.